xdmod/setup.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+echo "Import configuration files"
+if [ -f "/xdmod/conf/xdmod_etc.tar.gz" ]; then
+  tar -xvzf /xdmod/conf/xdmod_etc.tar.gz --strip-components 2 -C /etc/xdmod
+  chown -R root:apache /etc/xdmod
```

Comment:
The permissions are not the same on each file - most are owner xdmod group apache, some are root:xdmod. This will need to be updated before final release